tests/sles4sap/qesapdeployment/test_mirror.pm

```diff
@@ -16,12 +16,12 @@ sub run {
     my $provider_setting = get_required_var('PUBLIC_CLOUD_PROVIDER');
 
     if ($provider_setting eq 'AZURE') {
-        if (get_var("QESAPDEPLOY_IBSMIRROR_RESOURCE_GROUP")) {
-            my $rg = qesap_az_get_resource_group();
-            my $ibs_mirror_rg = get_var('QESAPDEPLOY_IBSMIRROR_RESOURCE_GROUP');
-            qesap_az_vnet_peering(source_group => $rg, target_group => $ibs_mirror_rg);
-            qesap_add_server_to_hosts(name => 'download.suse.de', ip => get_required_var("QESAPDEPLOY_IBSMIRROR_IP"));
-            qesap_az_vnet_peering_delete(source_group => $rg, target_group => $ibs_mirror_rg);
+        if (get_var('QESAPDEPLOY_IBSM_VNET') && get_var('QESAPDEPLOY_IBSM_RG')) {
+            my @remote_cmd = (
+                'ping -c3 ' . get_required_var('QESAPDEPLOY_DOWNLOAD_HOSTNAME'),
+                'zypper -n ref -s -f',
```

Comment:
been picky these two lines are already in another test module that is in the same schedule https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/0e0c757ff200d53b771f774e9e16a571fcf442cb/tests/sles4sap/qesapdeployment/test_system.pm#L27

But having them not executed there means to have the L19 if also there.